### PR TITLE
Update python.props to release python-worker v3

### PR DESCRIPTION
**Pull request by vrdmr**
Release info https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.2

### CODE REVIEW

**Review by ankitkumarr — DISMISSED** [review]

**Review by Hazhzeng — DISMISSED** [review]

File: build/python.props [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="1.1.202005032" />
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="3.0.13905" />
```
Comment (fabiocav) [inline build/python.props:3]: Are making a major version bump here?

Comment (pragnagopa) [reply build/python.props:3]: Assuming this is the new build generating nuget package with following semver semantics?

Comment (fabiocav) [reply build/python.props:3]: Followed up offline. We can address later.

Comment (fabiocav) [reply build/python.props:3]: @pragnagopa it doesn't match the release tags for the actual worker. This is artificially aligning with the host

**Review by Hazhzeng — APPROVED** [review]

**Pull request merged**